README.md

```diff
@@ -227,3 +227,12 @@ about the system requirements to support this mode of operation can be found [he
 
 EOF
 ```
+
+## Limitations
+### RDMA workloads utilizing RDMA Connection Manager (CM)
+
+When RDMA isolation is enabled, pods creation/deleteion can't be in parallel.
+This due to the face that moveing RDMA device to namespace and rebinding
+diffrent VF will rename RDMA character devices under /dev/infiniband.
+This can cause mismatch between the RDMA character devices and RDMA
+and device passed to the POD.
```

Comment:
nit: add "Causing traffic failue." at the end so the result of creating/destroying stuff in parallel is clear